eng/tools/generator/cmd/v2/common/generation.go

```diff
@@ -561,18 +561,17 @@ func (ctx *GenerateContext) GenerateForTypeSpec(generateParam *GenerateParam) (*
 			return nil, err
 		}
 
+		baseModule := fmt.Sprintf("%s/%s/%s", MgmtSDKModulePrefix, generateParam.RPName, generateParam.NamespaceName)
 		if _, err := os.Stat(filepath.Join(packagePath, "fake")); !os.IsNotExist(err) && oldModuleVersion.Major() != version.Major() {
 			log.Printf("Replace fake module v2+...")
-			if err = replaceModuleImport(packagePath, generateParam.RPName, generateParam.NamespaceName, oldModuleVersion.String(), version.String(),
-				"fake", ".go"); err != nil {
+			if err = ReplaceModule(version, packagePath, baseModule, ".go"); err != nil {
```

Comment:
Did we deprecate `replaceModuleImport` function?

Reply:
Using the improved method